launch/launch.go

```diff
@@ -33,6 +34,18 @@ func (m Metadata) FindProcessType(pType string) (Process, bool) {
 	return Process{}, false
 }
 
+func (m Metadata) FindLastDefaultProcessType() (Process, bool) {
+	defaultFound := false
+	var lastDefaultProcess Process
+	for _, p := range m.Processes {
+		if p.Default {
+			lastDefaultProcess = p
+			defaultFound = true
+		}
+	}
+	return lastDefaultProcess, defaultFound
+}
```

Comment:
Is this function needed anymore, now that we have `Metadata.Default`?

Reply:
No, we need to remove it.
Thanks!

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/520/commits/7b24df17bb72f57cfba08d780795cbaba5aaddca.